.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-    - "node"
```

Comment:
Why is this needed?

Reply:
For fixing travis errors.
But @sjro gave an answer. I will fix it again.